docker-compose.yaml

```diff
@@ -23,10 +23,10 @@ services:
         image: extrawurst/ip2country:latest
         ports:
             - 8854:5000
-    server:
+    backend:
+        image: ghcr.io/amsterdam-music-lab/muscle-backend:latest
         build:
```

Comment:
No, it will only push during the GHA. Locally, the image will be pulled from the registry, but when building anything that isn't cached (e.g., after changing requirements), it will use the `ghcr.io...` tag locally but not push.